include/clang/AST/Expr.h

```diff
@@ -2798,7 +2798,9 @@ class BoundsExpr : public Expr {
 /// classes).
 class CastExpr : public Expr {
 private:
-  enum { OP, BOUNDS, END_EXPR = 2 };
+  // BOUNDS - enum value for bounds of cast expression
+  // SUBBOUNDS - enum value for bounds of subexpression
```

Comment:
Could use the name SUBEXPR_BOUNDS instead of SUBBOUNDS?    The term SUBBOUNDS is ambiguous.  It could also mean a subrange of the bounds?